test/web3.js/SimpleCoin.js

```diff
@@ -18,6 +18,7 @@ describe('SimpleCoin', function () {
 
     deployerAddr = await getDeployerAddress()
   })
+  // FIXME: this doesn't work now
```

Comment:
Same ask for the short explanation 🙏

Reply:
same reason as the other - related to an RPC transaction receipt but I have no context to add to the code or ticket without digging deeper into the test